packages/credentials/tests/jwt.spec.ts

```diff
@@ -72,6 +82,113 @@ describe('Jwt', () => {
   });
 
   describe('verify()', () => {
+
+    let fetchStub: sinon.SinonStub;
+
+    beforeEach(() => {
+      // Setup stub so that a mocked response is returned rather than calling over the network.
+      fetchStub = sinon.stub(globalThis as any, 'fetch');
+
+      // By default, return a 200 OK response when fetch is called by publish().
+      fetchStub.resolves(fetchOkResponse());
+    });
+
+    afterEach(() => {
+      fetchStub.restore();
+    });
+
+    it('successful verify with did:dht', async () => {
+      const hexString =
+        '0ab2b3386e22595e1271e7ef67fda70c37acf7d28b8c884a6fdcbb0ea739f341' +
```

Comment:
Not a fan of opaque inputs to tests. Can you update and enable vector tests accordingly for the js implementation and call out in https://github.com/TBD54566975/web5-js/issues/638  that the kid changes must be reverted when 638 is fixed.

I think they should cover this change. Let me know if I am not making sense!

Reply:
This test tests the Jwt.sign / verify with did dht, (the reason we had to change the spec a bit in the first place)

Reply:
Got it, assuming the main intent of this test is to test sign and verify, can we avoid using the opaque hex string in the test since it is besides the point?